src/main/java/seedu/momentum/model/timer/Time.java

```diff
@@ -0,0 +1,67 @@
+package seedu.momentum.model.timer;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.momentum.commons.util.AppUtil.checkArgument;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeParseException;
+
+/**
+ * Represents a WorkDuration's time in the project book.
+ * Guarantees: immutable; is valid as declared in {@link #isValidTime(String)}
+ */
+public class Time {
+
+    public static final String MESSAGE_CONSTRAINTS = "Times should be in ISO8601 format";
```

Comment:
Will it be better to give an example of how the ISO8601 time format looks like in the error message? A user might not know what this format is.

Reply:
A user will never see this message (as of now) since we don't require users to manually input times. If we add commands that involve time inputs, then it might be better to not use ISO8601 format in the first place (its hard to read) and instead use some variation of the "yyyy MM dd hh:mm:ss" style.

This error only come up in logs when theres an error in loading saved data (e.g. the saved data is not in ISO format). For now I'll add an example of what the format looks like to the error message